config/application.rb

```diff
@@ -22,8 +22,14 @@ class Application < Rails::Application
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
     config.active_record.raise_in_transactional_callbacks = true
-    
+
     # Gzip files when possible
     config.middleware.use Rack::Deflater
+
+    #HCaseflow application urls
+    config.app_urls = {"Caseflow" => "https://caseflow.ds.va.gov", "Efolder" => "https://efolder.cf.ds.va.gov"}
```

Comment:
I'm partial to using a string here. I think the length it adds to the URL is negligible, and the readability is a lot better. It will be annoying for each app to have to know their own integer "Feedback ID". 

also, the case on efolder is "eFolder", never "Efolder".